documentation/modules/ROOT/pages/integrations/tracing.adoc

```diff
@@ -9,36 +9,103 @@
+
+* https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-api/[opentelemetry-api]
+* https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-context/[opentelemetry-context]
+* https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-semconv/[opentelemetry-semconv]
+* https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-instrumentation-api/[opentelemetry-instrumentation-api]
+* https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-instrumentation-api-semconv/[opentelemetry-instrumentation-api-semconv]
+* https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-kafka-clients-2.6/[opentelemetry-kafka-clients-2.6]
+* https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-kafka-clients-common/[opentelemetry-kafka-clients-common]
+ +
+To ensure version compatibility among the different OpenTelemetry packages, use Maven to download the packages.
+
+2. Create a POM file similar to the one in the following example.
+In your version of the file, specify the version of the OpenTelemetry package that you want to use.
+
+[source,xml]
+----
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+ <modelVersion>4.0.0</modelVersion>
+ <groupId>group.id</groupId>
+ <artifactId>artifact-id</artifactId>
+ <version>0.0.1</version>
+ <properties>
+  <version.opentelemetry.api>1.23.1</version.opentelemetry.api>
+  <version.opentelemetry.kafka.clients.2.6>1.23.0-alpha</version.opentelemetry.kafka.clients.2.6>
+ </properties>
+ <dependencies>
+  <dependency>
+   <groupId>io.opentelemetry</groupId>
+   <artifactId>opentelemetry-api</artifactId>
+   <version>${version.opentelemetry.api}</version>
+  </dependency>
+  <dependency>
+   <groupId>io.opentelemetry.instrumentation</groupId>
+   <artifactId>opentelemetry-kafka-clients-2.6</artifactId>
+   <version>${version.opentelemetry.kafka.clients.2.6}</version>
+  </dependency>
+ </dependencies>
+</project>
+----
+
+3. In the directory that contains the `pom.xml` file, run the following command to download all dependencies (including transitives) to a specified directory (for example. `./lib`)
 
-The tracing support is added to Debezium through the https://opentracing.io/[OpenTracing] specification.
-It is also necessary to provide a client implementing the specification.
-Debezium was tested with https://www.jaegertracing.io/[Jaeger] implementation.
+[source,bash]
+----
+mvn dependency:copy-dependencies -DoutputDirectory=./lib
+----
 
-[WARNING]
+==== Installing the SDK
+
+Install the OpenTelemetry SDK on your Kafka connect cluster by using the https://opentelemetry.io/docs/instrumentation/java/automatic/[Java agent] for automatic instrumentation.
+
+=== Use the {prodname} Kafka Connect image
+
```

Comment:
@roldanbob 

There are three different ways to install the Opentelemetry API and SDK. 

The first is completely free (manual installation) and allows the user to choose their own Kafka distribution, this is my case because I use the Confluent distribution. 

The second is a simplified way managed by Debezium with the Kafka connect image offered directly by Debezium. Personally I think this is useful for testing or prototyping only. 

The third highlights the Strimzi distribution which includes OpenTelemetry. It is intended for users who have chosen to use Strimzi on Kubernetes. personally I don't use it, but the Debezium team wanted plug and play integration with this distribution.